### DISCUSSION TITLE
[Issue] Difficult to install

### DISCUSSION THREAD

**Discussion by AP2020C** [Voltstro-Studios/UnityWebBrowser#340] [Q&A]
### What platform are you experiencing this issue on?

Windows x64

### What version of UWB are you using?

none

### What Unity version are you running?

2022.3.45

### Describe what the issue you are experiencing is.

The installation procedure first requires you to install VoltUPM, I tried, and failed. I have asked for support and hope they will answer.
However, usually Unity assets are downloaded directly from the asset store or by downloading the repositpry from GitHub, could it be that there is not a faster (and easier, thus reducing the possibility of error) way to get “Unity Web Browser”?

### Provide reproducible steps for this issue.

I tried to follow the procedure to download the asset, but failed

**Comment by Voltstro** [comment]
We don't have installation through Git since UWB has some large binaries that need to be included, and here on GitHub the regular limit is 100mb. Plus tracking binary files in Git is a bad idea.

We don't offer UWB on the Unity asset store as I personally hate the way the Unity asset store delivers libraries by just importing the library into your project. Fortunately Unity created the Unity Package Manager for solving this issue, however unfortunately Unity don't offer a way for the community to provide packages through an officially hosted registry, so have to resort to hosting a registry myself.
